app/src/main/java/org/fossasia/openevent/app/qrscan/ScanQRActivity.java

```diff
@@ -255,4 +189,39 @@ public void showToggleDialog(long attendeeId) {
             presenter.resumeScan();
         });
     }
+
+    /**
+     * Creates and starts the camera.
+     */
+    private void createCameraSource() {
+        Context context = getApplicationContext();
+        barcodeDetector = new BarcodeDetector.Builder(context).build();
+        BarcodeTrackerFactory barcodeFactory = new BarcodeTrackerFactory(graphicOverlay);
```

Comment:
See this https://medium.com/@froger_mcs/inject-everything-viewholder-and-dagger-2-e1551a76a908